### PR TITLE
Closes #271 | Implement dataloader for UiT-ViCTSD

### LINKED ISSUE #271: Create dataset loader for UIT-ViCTSD
Dataloader name: `uit_victsd/uit_victsd.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?uit-victsd

|     Dataset| uit_victsd |
|-------------|---|
| Description | The UIT-ViCTSD (Vietnamese Constructive and Toxic Speech Detection dataset) is a compilation of 10,000 human-annotated comments intended for constructive and toxic comments detection.  Divided into three parts, it includes a Training set with 7,000 comments, a Validation set containing 2,000 comments, and a Test set with 1,000 comments. The composition of the dataset spanning 10 domains, reflecting the diverse expressions found in social media interactions among Vietnamese users. |
| Subsets | constructiveness, toxicity |
| Languages | vie |
| Tasks | Text Classification |
| License | Unknown (unknown) |
| Homepage | https://github.com/tarudesu/ViCTSD |
| HF URL | https://huggingface.co/datasets/tarudesu/ViCTSD |
| Paper URL | https://link.springer.com/chapter/10.1007/978-3-030-79457-6_49 |

### PR DESCRIPTION
**Pull request by elyanah-aco**
Closes #271.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Review by jen-santoso — APPROVED** [review]
Hello @elyanah-aco 
Thanks for the implementation. Tested and runs well, LGTM. 
Let's wait for the other reviewer

**Review by ljvmiranda921 — APPROVED** [review]
LGTM! Merging in a few 👍 Thanks for the nice PR!
